src/meter/MeterState.ts

```diff
@@ -0,0 +1,9 @@
+import {
+  UseProgressProps,
+  useProgressState,
+  ProgressStateReturn,
+} from "../progress";
+
+export type UseMeterProps = UseProgressProps;
+export const useMeterState = useProgressState;
+export type MeterStateReturn = ProgressStateReturn;
```

Comment:
Re-exporting the component and use them in storybook result in some warning.
I had to create a file and re export individually to remove those warnings.

Reply:
Okay i guess we can left it as it is.